### ISSUE TITLE
Installer sh script logging `/usr/bin/ldd: line 41: printf: write error: Broken pipe` when run

### ISSUE THREAD

**Issue by adriencaccia** [axodotdev/cargo-dist#624]
Hey, when executing the installation script generated by `cargo-dist` on a GitHub Actions runner, the following is being logged:

```txt
/usr/bin/ldd: line 41: printf: write error: Broken pipe
/usr/bin/ldd: line 43: printf: write error: Broken pipe
```

You can see it in action here: https://github.com/PyO3/pyo3/actions/runs/7093258442/job/19306303498?pr=3627#step:8:52

It pollutes the output log unnecessarily, but It does not impact the execution of the script though.

I was able to narrow it down to the following line:
https://github.com/axodotdev/cargo-dist/blob/41e79b10533c10c85a0c0bc5fb556e096c2c6357/cargo-dist/templates/installer/installer.sh.j2#L515-L515

By doing some more testing, it seems that running `ldd --version | head -1` in a GitHub Actions runner will work fine but outputs some "Broken pipe" lines to stderr. See https://github.com/CodSpeedHQ/action/actions/runs/7099719565/job/19324314497?pr=90#step:3:439 for an example.

Is there a way to not have those anymore?

Thanks!

**Comment by Gankra** [comment]
Just taking a moment to say: this is deeply perplexing and we agree it should be fixed.

**Comment by mistydemeo** [comment]
Should be fixed by #627. The fix will be in cargo-dist 0.6.0.

**Comment by adriencaccia** [comment]
Wow, thank you so much for the quick response and fix 🤩
